### DISCUSSION TITLE
Image spams

### DISCUSSION THREAD

**Discussion by flaviozluca** [rspamd/rspamd#5197] [Q&A]
Friends!

Check out the false negative. Its overall an image. Shouldnt it score more?

Tks!
[Uploading 'Expires soon  your PREDATOR 3500 Watt Inverter Generator reward' - PREDATOR 3500 Watt (service@pef.pelisplus3.ink) - 2024-10-23 1045.zip…]()

**Comment by flaviozluca** [comment]
this one too. we mark as spam with a score of 11, came close. but image proportion, indepedently of embedded images or not, perhaps it should score more.
['Reminder about your Medicare Kit reward' - CVS unlocked (supportt@xuk.papystreaming.team) - 2024-10-23 1405.zip](https://github.com/user-attachments/files/17508379/Reminder.about.your.Medicare.Kit.reward.-.CVS.unlocked.supportt%40xuk.papystreaming.team.-.2024-10-23.1405.zip)

**Comment by eosps** [comment]
Nowadays, many phishing emails convert information into images or convert URLs into QR codes to evade inspection. I wonder if there is any way to filter them.
